src/test/java/no/hvl/dat250/h2020/group5/service/GuestServiceTest.java

```diff
@@ -21,17 +26,49 @@
 
     @Mock GuestRepository guestRepository;
 
+    private Guest guest1;
+    private Guest guest2;
+    private Guest guest3;
+
+    @BeforeEach
+    public void setup() {
+
+        this.guest1 = new Guest();
+        this.guest1.setId((long) 1);
```

Comment:
Mulig å bruke "L" bak tallet i stedet for å caste det til long, f.eks: 1L

Reply:
Nice! Skal endre det 🤩